src/libais/ais.h

```diff
@@ -622,6 +622,7 @@ class Ais6_1_5 : public Ais6 {
   bool ai_available;  // TODO(schwehr): AI?  Is this the dac/fi being acked?
   int ai_response;
   int spare;
+  int spare2;
```

Comment:
This change is in from https://github.com/schwehr/libais/commit/65cb1b12d65ebab97a34f4e54daaae81eab4b949